src/screens/profile/components/navbar/constants.ts

```diff
@@ -0,0 +1,8 @@
+import { SCREEN_WIDTH } from 'src/common/constants';
+import { DEFAULT_SCREEN_PADDING } from 'src/styles';
+
+const NAVBAR_BUTTON_WIDTH = Math.ceil(
+  (SCREEN_WIDTH - DEFAULT_SCREEN_PADDING * 2) / 3
```

Comment:
Why do we need this? And what we'll do if there are more than 3 buttons or with long-name-button?